### ISSUE TITLE
Several Windows Bugs Still in 4.0 b2

### ISSUE THREAD

**Issue by seanocali** [andyduke/filesystem_picker#46]
Love this plugin, it's crucial for me as I wish to support Gamepad and Dpad remote support for a Windows app and the native picker doesn't allow this. There are still several bugs which prevent me from using it in production. I've attached a video showing them and the sample project.

[filesystem_test_project.zip](https://github.com/andyduke/filesystem_picker/files/10751512/filesystem_test_project.zip)

https://user-images.githubusercontent.com/16900871/219255302-cac960ed-1517-4096-a94f-7dc47643d837.mp4

**Comment by andyduke** [comment]
@seanocali Can you also provide the output of the `flutter doctor -v` command, as well as the version of your operating system?

**Comment by seanocali** [comment]
Windows 11 Pro
10.0.22000 Build 22000

```
[√] Flutter (Channel stable, 3.7.3, on Microsoft Windows [Version 10.0.22000.1574], locale en-US)
    • Flutter version 3.7.3 on channel stable at C:\Users\seano\source\repos\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 9944297138 (11 days ago), 2023-02-08 15:46:04 -0800
    • Engine revision 248290d6d5
    • Dart version 2.19.2
    • DevTools version 2.20.1

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at C:\Users\seano\AppData\Local\Android\sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.4.4)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.4.33213.308
    • Windows 10 SDK version 10.0.22000.0

[√] Android Studio (version 2022.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[√] VS Code (version 1.75.0)
    • VS Code at C:\Users\seano\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.1574]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 110.0.5481.96
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 110.0.1587.41

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```

**Comment by andyduke** [comment]
Fixed in 4.0.0-beta.3
